zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -342,9 +344,16 @@ def __attrs_post_init__(self):
                     "`intermediaries_dir` must be specified when `force_intermediaries`==True."
                 )
 
-        self.processing_chunker = VolumetricIndexChunker(
-            chunk_size=self.processing_chunk_size, resolution=self.dst_resolution
-        )
+        if self.gap is not None and self.gap != Vec3D[int](0, 0, 0):
+            self.processing_chunker = VolumetricIndexChunker(
+                chunk_size=self.processing_chunk_size,
+                resolution=self.dst_resolution,
+                stride=self.processing_chunk_size + self.gap,
+            )
+        else:
+            self.processing_chunker = VolumetricIndexChunker(
+                chunk_size=self.processing_chunk_size, resolution=self.dst_resolution
+            )
```

Comment:
We don't want to have `is not None` checks everywhere - see what's done for `self.roi_crop_pad`, `self.processing_blend_pad`, etc above: `self.gap` can just be set to `Vec3D[int](0, 0, 0)`, and then this if-else block can be removed.